test/run

```diff
@@ -13,6 +13,16 @@ if [ "$TEST_SCENARIO" == "devel" ]; then
     export TEST_COVERAGE=yes
 fi
 
+if [ "$TEST_SCENARIO" == "podman-next" ]; then
+    export TEST_COPR="rhcontainerbot/podman-next"
+fi
+
+TEST_COPR=${TEST_COPR:-}
```

Comment:
This looks odd and unnecessary. It's better to not set the variable at all than to an empty value.